src/orderbook/index.ts

```diff
@@ -129,15 +180,74 @@ export class StreamedOrderbook {
         fromBlock + this.options.blockPageSize - 1,
         endBlock,
       )
-      console.error(`fetching page ${fromBlock}-${toBlock}`)
 
+      console.error(`fetching page ${fromBlock}-${toBlock}`)
```

Comment:
just noticed that these (and some other new non error messages in this PR) are printed to error. Is there a reason for that?

Reply:
Just to print to stderr - these will be cleaned up.